src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { People } from './components/People/People';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<h1 className="title">Home Page</h1>} />
+          <Route path="home" element={<Navigate to="/" />} />
```

Comment:
The `Navigate` component is used correctly for redirecting from `/home` to `/`. Ensure that this behavior is intended and that there are no other routes that should be handled differently.